interview-questions.md

```diff
@@ -7,62 +7,62 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-  Your answer:
+  Your answer: A conditional statement is a process that evaluates the input/output through a series of possible outcomes providing an action based on whether or not the condition was met
 
-  Researched answer:
+  Researched answer: Conditional statements are used to determine the behavior of a block of code  based on IF a condition is met, or ELSE something will happen or something ELSE will happen IF another specific scenario occurs.
 
 
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: Git is a version tracking and version control for local repositories, Github is the online cloud that utilizes Git to perform remote to local version tracking and control for repositories. 
 
-  Researched answer:
+  Researched answer: Git is an open source version control system created by the inventor of Linux. Git provides the user with a complete local repository tha
 
 
 
 2. Which JavaScript operators will return a Boolean value?
 
-  Your answer:
+  Your answer: Logical operators such as "&&" " || " and " ! " will return a value of true or false 
 
-  Researched answer:
+  Researched answer: Logical operators return a value of truthy or falsy. Examples of these operators are AND (&&), OR (||), and NOT (!) 
```

Comment:
Great!  There are also Relational (>, <, <=, >=) and Equality (==, ===) operators that will return a boolean value.